CHANGES.md

```diff
@@ -3,6 +3,7 @@ Unreleased
 
 - Fix preludes not being recorded as dependencies in the `(mdx)` stanza (#7109,
   fixes #7077, @emillon).
+- Make coq_db creation in scope lazy (@ejgallego, #7133)
```

Comment:
I am not sure this is going to be noticeable for users.

Reply:
Hopefully it is not.

Reply:
To clarify why I added the changelog, as this is a large change in (internal) behavior, Dune Coq users appreciate having a list of things to look at if their setup breaks at upgrade time.